### DISCUSSION TITLE
Network Analytics Page for WP Multisite

### DISCUSSION THREAD

**Discussion by dannyvankooten** [ibericode/koko-analytics#201] [Ideas]
Quoting @beckej13820 in #172:

> I wish there was a network analytics page that could give some summary statistics on the entire network of WordPress Sites.
> 
> I'd like to be able to:
> 
>     See overall traffic on all sites
>     Instead of seeing which pages have the most traffic, see which subsites have the most traffic
>     Be able to click the links in the report and go directly to a single sites analytics report

**Comment by beckej13820** [comment]
Thanks for preserving this as you move to a different system!
